google/cloud/forseti/notifier/notifiers/email_violations.py

```diff
@@ -18,12 +18,12 @@
 
 from google.cloud.forseti.common.data_access import csv_writer
 from google.cloud.forseti.common.util import date_time
-from google.cloud.forseti.common.util import email
 from google.cloud.forseti.common.util import errors as util_errors
 from google.cloud.forseti.common.util import logger
 from google.cloud.forseti.common.util import parser
 from google.cloud.forseti.common.util import string_formats
 from google.cloud.forseti.notifier.notifiers import base_notification
+from google.cloud.forseti.common.util.email import email_factory
```

Comment:
alpha sort

Reply:
Done.

Reply:
Had to regroup imports to fix pylint error: Imports from package google are not grouped (ungrouped-imports) and Imports from package sendgrid are not grouped (ungrouped-imports).